tech-specs/multi-proof-and-verifiable-compute/README.md

```diff
@@ -4,7 +4,11 @@
 
 <figure><img src="../../.gitbook/assets/Intro-Multi-Proof.png" alt=""><figcaption></figcaption></figure>
 
-ZK is the pronoun of verifiable computing of today’s Web3, however, everyone should admit that Zero Knowledge is still early even the innovation happens extremely fast every day, and the performance issue will exist in the visible future. But we don’t limit ourselves to Zero Knowledge and use single proof in our design, other technologies like TEE, FHE, and MPC also should be considered. The following sections introduce the multi-proof system built on top of the Phala blockchain. By having this guide, we aim to elaborate on the necessity of multi-proof in the blockchain industry, and how we build it for developers to easily integrate TEE execution proof into their application.
+In the blockchain space, our vision is to bring privacy, security, and verifiability to users. Our ultimate goal is to design a system that can simultaneously fulfill these attributes. However, relying on a single proof system to achieve this is impractical for several reasons:
+
+- No single cryptographic system can be guaranteed to be 100% secure. For instance, zero-knowledge proofs (zk) can have soundness bugs that are difficult to detect, while Multi-Party Computation (MPC) is susceptible to collusion risks between nodes.
```

Comment:
![image](https://github.com/user-attachments/assets/ed07fc8a-a3bc-4498-8400-20469d0457b2)
This looks odd in docs version. Is there a reason to have a mix of - and number bullet points?

Reply:
Updated